tests/pagila/copydb.sh

```diff
@@ -45,6 +45,15 @@ create schema "With a space";
 create extension seg schema "With a space" cascade;
 EOF
 
+# Create sequences and tables to ensure they get reset.
+psql -d ${PAGILA_SOURCE_PGURI} <<EOF
+create sequence normal_table_id_seq;
+create table normal_table (id integer primary key default nextval('normal_table_id_seq'));
+create table identity_table (id integer primary key generated always as identity);
+select setval('identity_table_id_seq', 667);
+select setval('normal_table_id_seq', 667);
+EOF
+
```

Comment:
Can we move this to the tests/unit suite? I think it belongs there.

Reply:
Yep yep, done.